nion/swift/Inspector.py

```diff
@@ -3195,12 +3195,14 @@ def __create_position_and_shape_ui(self) -> Declarative.UIDescriptionResult:
             return self.__shape_and_pos_func()
 
 
-class GraphicsCalibrationHandler(Declarative.Handler):
-    def __init__(self, document_controller: DocumentController.DocumentController, display_item: DisplayItem.DisplayItem):
-        super().__init__()
-
+class GraphicsSectionHandler(Declarative.Handler):
+    def __init__(self, document_controller: DocumentController.DocumentController,
+                 display_item: DisplayItem.DisplayItem,
+                 graphics_model: ListModel.ObservedListModel[DisplayItem.DisplayLayer]):
```

Comment:
This needs a `super().__init__()` to properly initialize `Declarative.Handler`, which itself has to initialize `Observable`.